tests/components/cloud/__init__.py

```diff
@@ -4,13 +4,13 @@
 from homeassistant.components import cloud
 from homeassistant.components.cloud import const
 from homeassistant.setup import async_setup_component
-
 from tests.common import mock_coro
 
 
 async def mock_cloud(hass, config=None):
     """Mock cloud."""
-    assert await async_setup_component(hass, cloud.DOMAIN, {"cloud": config or {}})
+    assert await async_setup_component(hass, cloud.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)